mongo/integration/client_side_encryption_test.go

```diff
@@ -4,6 +4,7 @@
 // not use this file except in compliance with the License. You may obtain
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
+//go:build cse
```

Comment:
Was this intended to be included?

Regardless, adding `go:build cse` seems OK to add based on https://stackoverflow.com/a/68361186/774658.